File: CapsValidationProyect.Persistence/DapperConnection/Employee/EmployeeModel.cs
using System;

namespace CapsValidationProyect.Persistence.DapperConnection.Employee
{
    public class EmployeeModel
    {
   public int Id { get; set; }

    public int DepartmentId { get; set; }

    public string FullName { get; set; } = null!;

    public string FirstName { get; set; } = null!;

    public string MiddleName { get; set; } = null!;

    public string LastName { get; set; } = null!;

    public string MothersLastName { get; set; } = null!;

    }
}

File: CapsValidationProyect.NuxtClient/Entities/Employee/EmployeeModel.ts

export interface IEmployeeModel {
    id: number;
    departmentId: number | null;
    fullName: string;
    firstName: string;
    middleName: string;
    lastName: string;
    mothersLastName: string;
    department: string;
  }

export class EmployeeModel implements IEmployeeModel {
    id: number;
    departmentId: number | null;
    fullName: string;
    firstName: string;
    middleName: string;
    lastName: string;
    mothersLastName: string;
    department: string = '';
  
    constructor(
      id: number = 0,
      departmentId: number | null = null,
      fullName: string = '',
      firstName: string = '',
      middleName: string = '',
      lastName: string = '',
      mothersLastName: string = ''
    ) {
      this.id = id;
      this.departmentId = departmentId;
      this.fullName = fullName;
      this.firstName = firstName;
      this.middleName = middleName;
      this.lastName = lastName;
      this.mothersLastName = mothersLastName;
    }
  }